addons/project_issue/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,70 @@
+---Fields in module 'project_issue'---
+project_issue / project.issue            / categ_ids (many2many)         : DEL relation: project.category
+project_issue / project.issue            / tag_ids (many2many)           : NEW relation: project.tags
+# TODO DONE
+
+project_issue / project.issue            / kanban_state (selection)      : now required, default = normal
+# Nothing to do
+
+project_issue / project.issue            / message_follower_ids (many2many): not a function anymore
+project_issue / project.issue            / message_follower_ids (many2many): relation is now 'mail.followers' ('res.partner')
+project_issue / project.issue            / message_follower_ids (many2many): type is now 'one2many' ('many2many')
+# Concerns module Mail (has to be done in another migration script specifically for that module) ---> Nothing to do
```

Comment:
rephrase as "Done in mail migration script", so that this sentence serves in the future.